### DISCUSSION TITLE
[New Connector]: SurrealDB

### DISCUSSION THREAD

**Discussion by PierreSnell** [airbytehq/airbyte#36112] [New Connector Request]
## Tell us about the new connector you’d like to have
* Which source and which destination? 
Destination: Surreal (https://surrealdb.com/)
* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB?  
No

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
We use the Surreal database to get more easily exploitable data
* How often do you want to run syncs?
X hours or X days is fine.
* If this is an API source connector, which entities/endpoints do you need supported?
You can use HTTP / Websocket or clients : https://surrealdb.com/docs/integration
* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name? 
You can host it / open-source : https://surrealdb.com/docs/start/installation
Or sign-up for the beta-cloud : https://surrealdb.com/docs/start/installation

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
Create a custom connector and maintain it...

## Are you willing to submit a PR?

Yes if the connector is not here we would have to do our own and could do a PR. But as we're not experts (for now ;) ) with airbyte.

**Comment by mysticaltech** [comment]
Super needed, I could have used this right now. Too bad it's not done already.

[a comment by an automated account is not shown]
